tmsconnect.php

```diff
@@ -102,5 +167,30 @@ function tmsc_get_baseurl() {
  * Enqueue scripts and styles
  */
 function tmsc_enqueue_assets() {
+	wp_enqueue_script( 'tmsc-admin-sync', tmsc_get_baseurl() . '/js/tmsc-admin-sync.js', array( 'jquery' ), '0.1' ,true );
+	wp_enqueue_script( 'tmsc-admin-custom-landings', tmsc_get_baseurl() . '/js/tmsc-admin-custom-landings.js', array( 'jquery' ), '0.1' ,true );
+	wp_localize_script( 'tmsc-admin-sync', 'tmscAdminData', array( 'wp_admin_nonce' => wp_create_nonce( 'wp_admin_js_script' ), 'wp_ajax_url' => esc_url_raw( admin_url( 'admin-ajax.php' ) ) ) );
 }
 
+/**
+ * Register a migrating post status.
+ */
+function tmsc_register_migrating_status() {
```

Comment:
Not a blocker. This might go well in a `inc/posts.php` file or something like that.